sql/core/src/test/resources/sql-tests/inputs/udf/pgSQL/udf-aggregates_part1.sql

```diff
@@ -23,32 +23,32 @@ select CAST(avg(udf(b)) AS Decimal(10,3)) AS avg_107_943 FROM aggtest;
 -- `student` has a column with data type POINT, which is not supported by Spark [SPARK-27766]
 -- SELECT avg(gpa) AS avg_3_4 FROM ONLY student;
 
-SELECT sum(udf(four)) AS sum_1500 FROM onek;
+SELECT CAST(sum(udf(four)) AS int) AS sum_1500 FROM onek;
```

Comment:
I guess only float/double has the issue, we cast int/long just for more robust too?

Reply:
Ah, we don't have to. But casting makes closer to the original results:

```diff
 -- !query 3
-SELECT sum(four) AS sum_1500 FROM onek
+SELECT CAST(sum(udf(four)) AS int) AS sum_1500 FROM onek
 -- !query 3 schema
-struct<sum_1500:bigint>
+struct<sum_1500:int>
 -- !query 3 output
 1500
```

If we don't cast, it returns a double due to type coercion. Should be fine.